source/src/components/html_fragments/TaskPopUp.html

```diff
@@ -0,0 +1,131 @@
+<style>
+    .button-footer {
+        background-color: rgb(234 234 234);
+        /* padding: 14px 20px; */
+        padding: 1.09375vw 1.5625vw;
+        text-align: right;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        left: 0;
+        /* border-bottom-left-radius: 4px; */
+        /* border-bottom-right-radius: 4px; */
+        border-bottom-left-radius: 0.3125vw;
+        border-bottom-right-radius: 0.3125vw;
+    }
+    #close-icon {
+        /* width: 15px; */
+        width: 1.171875vw;
+        /* margin-top: 10px; */
+        /* margin-right: 10px; */
+        margin-top: 0.78125vw;
+        margin-right: 0.78125vw;
+        position:absolute;
+        top:0;
+        right:0;
+        cursor: pointer;
+        opacity: 0.33;
```

Comment:
The smoothened animation code is not updated (because my pull request did not go through). I'll add them back after fetching the changes.

Reply:
+1